modules/user/user.repository.js

```diff
@@ -0,0 +1,43 @@
+const express = require("express");
```

Comment:
Is express used anywhere in this file?

Reply:
No, deleted express from user.repository.js